cli/commands/migrate_create.go

```diff
@@ -111,12 +119,21 @@ func (o *migrateCreateOptions) run() (version int64, err error) {
 	if o.flags.Changed("metadata-from-file") && o.sqlServer {
 		return 0, errors.New("only one sql type can be set")
 	}
+
 	if o.flags.Changed("metadata-from-file") && o.metaDataServer {
 		return 0, errors.New("only one metadata type can be set")
 	}
 
+	if o.flags.Changed("up-sql") && !o.flags.Changed("down-sql") {
```

Comment:
Hey, @kolharsam extremely sorry, I messed up the previous suggestion, I actually meant the suggestion to be in this line, looks like it was misplaced. The Idea was to, allow a user to create a migration with either an up migration or a down migration but show a warning. Can you please make this change?

Reply:
@scriptonist got it. I'll revert the previous suggestions and make this less strict I guess, since we only have to display the warnings.